cli/src/account.rs

```diff
@@ -94,7 +96,7 @@ pub async fn account_prompt_internal(
                 }
                 _ => {
                     // Prepend `Account: ` so the parsing will be correct
-                    let command = format!("Account: {}", command.trim());
+                    let command = format!("Account: {}", command);
```

Comment:
```suggestion
                    let command = format!("Account: {command}");
```
Why doesn't clippy show this?

Reply:
Not sure why it doesn't 😢